### PR TITLE
Pending carbon credits

### LINKED ISSUE #40: Display Pending Carbon  Credits on Admin
-Display pending carbon credits on Admin//PendingCarbonCredits.js
-Api to fetch from is api/v1/carbon_credits

### LINKED ISSUE #44: Clean Up Styling
-Clean up the Styling of the project

### PR DESCRIPTION
**Pull request by AfandiElomda**

### CODE REVIEW

File: src/components/Admin/PendingCarbonCredits.js [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+import './PendingCarbonCredit.css';
+import PendingCarbons from './PendingCarbons';
+
+function PendingCarbonCredits({loggedInUserId}) {
+
+  const [pendingCard, setPendingCard] = useState([]);
+
+  useEffect(() => {
+    fetch(`/api/v1/carbon_credits/${loggedInUserId}`, {
+      method: "GET",
```
Comment (MICHAELMUNAVU83) [inline src/components/Admin/PendingCarbonCredits.js]: -Fetch from "api/v1/carbon_credits as we want to have all carbon credits

File: src/components/Admin/PendingCarbons.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <div className="Card">
+        <img
+          src="https://images.unsplash.com/photo-1503785640985-f62e3aeee448?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1174&q=80"
+          alt="image"
+        /> 
+        <p className="p1"><Link>{carbonCredit.source}</Link></p>
+        <p className="p2">  At a price of ${carbonCredit.price}, 10,000 {carbonCredit.source} planted
+           on 2 acres of land.</p>
+        <p  className="p3">Offsetting {carbonCredit.amount} of C02</p>
+        <p><button type="button" class="btn btn-success"> Approve</button></p>
```
Comment (MICHAELMUNAVU83) [inline src/components/Admin/PendingCarbons.js]: -center this button and change its color to green

**Review by MICHAELMUNAVU83 — APPROVED** [review]
Hi, @AfandiElomda 

Good job so far coming so far with the progress you've made! 💪💪

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️

 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

File: src/components/Admin/PendingCarbonCredits.js (hunk 1 above)
Comment (AfandiElomda) [reply src/components/Admin/PendingCarbonCredits.js]: corrected the errors

**Review by MICHAELMUNAVU83 — APPROVED** [review]
Hi @AfandiElomda 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- ✔ Followed figma design
- ✔ Components structure 

## Optional suggestions ☂

**Pull request merged**